examples/nextjs/app/ui/payment-orders/table.tsx

```diff
@@ -56,6 +57,24 @@ export default function PaymentOrdersTable({
                 minute: 'numeric'
               })}
             </td>
+            <td className="group/edit invisible whitespace-nowrap px-4 py-3 text-sm group-hover/item:visible">
```

Comment:
```suggestion
            <td className="group/edit opacity-0 whitespace-nowrap px-4 py-3 text-sm group-hover/item:opacity-100 transition-opacity">
```

In my opinion, these suggestions improve the user experience.